options/ansi/include/setjmp.h

```diff
@@ -12,7 +12,7 @@ extern "C" {
 
 // [C11/7.13] Non-local jumps
 
-typedef struct {
+typedef struct __jmp_buf {
```

Comment:
what's this used for?

Reply:
I honestly don't remember it but something needed it.